Formula/php.rb

```diff
@@ -0,0 +1,333 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net/"
+
+  stable do
+    version "7.1.10"
+    url "http://php.net/get/php-7.1.10.tar.gz/from/this/mirror"
+    sha256 "edc6a7c3fe89419525ce51969c5f48610e53613235bbef255c3a4db33b458083"
+
+    depends_on "libtool" => :run
+    depends_on "mcrypt"
+  end
+
+  devel do
+    url "https://downloads.php.net/~remi/php-7.2.0RC3.tar.gz"
+    sha256 "076491818dd80d723c9232bd4553a95eb30374f3c602d5ca67b95f2ac5e1655a"
+
+    depends_on "argon2"
+    depends_on "libsodium"
+  end
+
+  option "with-thread-safety", "Build with thread safety"
+
+  depends_on "aspell"
+  depends_on "curl" if MacOS.version < :lion
+  depends_on "enchant"
+  depends_on "freetds"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "httpd"
```

Comment:
Not sure how I feel about httpd being a dependency here. First of all, it requires additional work if you want to use a web server other than Apache. It also forces you to have a webserver installed even if you don't need it. Not knowing why it's a hard dependency, I'd ask to reconsider this decision.

Reply:
I don't want any options on this formula. So it's either in or it's out. My understanding is that this formula is more useful if it's in.

Reply:
"useful" is relative - I don't consider it useful to have a web server installed and running just because I installed PHP. PHP doesn't need a web server to run. Installing one by default is a bad choice IMO.

Reply:
You're entitled to your opinion, of course.